.github/workflows/ci-aarchxx.yml

```diff
@@ -160,7 +160,7 @@ jobs:
 
```

Comment:
> Upgrades Ubuntu version in AArchXX CI jobs on Github to 20.04. 

So the cross-compile ones that run tests on QEMU were already on 20.04, which I did to get the latest QEMU.  So it may be best to update this and maybe the title: this is only updating Android and a64-on-x86.

Reply:
I updated details in the description. But made only a small change in title, to keep it short.